domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/domene/vilkår/Vurdering.kt

```diff
@@ -1,8 +1,12 @@
 package no.nav.tiltakspenger.saksbehandling.domene.vilkår
 
+import no.nav.tiltakspenger.libs.periodisering.Periode
 import no.nav.tiltakspenger.saksbehandling.domene.saksopplysning.Kilde
 import java.time.LocalDate
 
+/**
+ * TODO jah: Forsvinner når vi har flyttet alt til Vilkår.
```

Comment:
Jeg er ikke 100% sikker på om Vurdering kan forsvinne, se kommentar under KvpVilkår

Reply:
Vi prøver uten Vurdering for Kvp. Den kommer sannsynligvis inn for de vilkårene som trenger det